docs/docs/themes/using-multiple-gatsby-themes.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
 ---
 
-Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+Gatsby temas están destinados a ser componible. Esto significa que puede instalar temas múltiples el uno junto al otro.
 
-The `gatsby-starter-theme` composes two Gatsby themes: `gatsby-theme-blog` and `gatsby-theme-notes`
+La `gatsby-starter-theme` compone dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
 
 ```shell
 gatsby new my-notes-blog https://github.com/gatsbyjs/gatsby-starter-theme
 ```
 
-The starter includes both theme packages (`gatsby-theme-blog` and `gatsby-theme-notes`) in the starter's `gatsby-config.js` file.
+El inicio incluye ambos paquetes de tema (`gatsby-theme-blog` y `gatsby-theme-notes`) en el archivo `gatsby-config.js` del starter.
```

Comment:
```suggestion
El starter incluye ambos paquetes de tema (`gatsby-theme-blog` y `gatsby-theme-notes`) en el archivo `gatsby-config.js` del starter.
```

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.